src/locale/de_DE.js

```diff
@@ -84,51 +84,51 @@ export const de_DE = {
   LBL_LEFT: 'Links',
   LBL_RIGHT: 'Rechts',
   LBL_BOTTOM: 'Unten',
-  LBL_MENU: 'Menu',
+  LBL_MENU: 'Menü',
   LBL_ERROR: 'Error',
   LBL_INFO: 'Info',
   LBL_MESSAGE: 'Message',
-  LBL_WARNINIG: 'Warning',
+  LBL_WARNINIG: 'Warnung',
   LBL_SUCCESS: 'Success',
   LBL_FAILURE: 'Failure',
-  LBL_WINDOW: 'Window',
-  LBL_WINDOWS: 'Windows',
-  LBL_NOTIFICATION: 'Notification',
-  LBL_NOTIFICATIONS: 'Notifications',
-  LBL_TRAY: 'Tray Entry',
+  LBL_WINDOW: 'Fenster',
+  LBL_WINDOWS: 'Fenster',
+  LBL_NOTIFICATION: 'Benachrichtigung',
+  LBL_NOTIFICATIONS: 'Benachrichtigungen',
+  LBL_TRAY: 'Tray-Einführung',
   LBL_NAME: 'Name',
-  LBL_TYPE: 'Type',
-  LBL_SIZE: 'Size',
-  LBL_FILE: 'File',
-  LBL_NEW: 'New',
-  LBL_OPEN: 'Open',
-  LBL_SAVE: 'Save',
-  LBL_SAVEAS: 'Save As',
+  LBL_TYPE: 'Typ',
+  LBL_SIZE: 'Format',
+  LBL_FILE: 'Datei',
+  LBL_NEW: 'Neue',
+  LBL_OPEN: 'Öffnen',
+  LBL_SAVE: 'Speichern',
+  LBL_SAVEAS: 'Speichern als',
   LBL_OK: 'OK',
-  LBL_ABORT: 'Abort',
-  LBL_CANCEL: 'Cancel',
-  LBL_CLOSE: 'Close',
-  LBL_QUIT: 'Quit',
+  LBL_ABORT: 'Abbrechen',
+  LBL_CANCEL: 'Beenden',
+  LBL_CLOSE: 'Schließen',
+  LBL_QUIT: 'Verlassen',
   LBL_YES: 'Ja',
   LBL_NO: 'Nein',
-  LBL_GO: 'Go',
-  LBL_MKDIR: 'Create new directory',
-  LBL_MKFILE: 'Create new file',
-  LBL_COPY: 'Copy',
-  LBL_PASTE: 'Paste',
-  LBL_CUT: 'Cut',
-  LBL_MOVE: 'Move',
-  LBL_RENAME: 'Rename',
-  LBL_DELETE: 'Delete',
-  LBL_DOWNLOAD: 'Download',
-  LBL_REFRESH: 'Refresh',
-  LBL_RELOAD: 'Reload',
-  LBL_HOME: 'Home',
-  LBL_VIEW: 'View',
+  LBL_GO: 'Los',
+  LBL_MKDIR: 'Neues Verzeichnis erstellen',
+  LBL_MKFILE: 'Neue Datei erstellen',
+  LBL_COPY: 'Kopieren',
+  LBL_PASTE: 'Einfügen',
+  LBL_CUT: 'Schneiden',
+  LBL_MOVE: 'Bewegen',
```

Comment:
'Verschieben'
While "Bewegen" is not wrong i guess "Verschieben" makes more sense.

Reply:
done. Yes it makes more sense